src/video-grid/useMediaStream.ts

```diff
@@ -213,7 +213,7 @@ export const useSpatialMediaStream = (
   const sourceRef = useRef<MediaStreamAudioSourceNode>();
 
   useEffect(() => {
-    if (spatialAudio && tileRef.current && !mute) {
+    if (spatialAudio && audioContext && tileRef.current && !mute) {
```

Comment:
audioContext is intended to always be present as per the type, so I don't think this is necessary?

Reply:
I've seen an issue where it is not :/ Might be related to my SFU branch - will try to track it down and send a fix if it is relevant for regular EC